services/gcn_service/gcn_service.py

```diff
@@ -142,7 +142,8 @@ def poll_events(*args, **kwargs):
                         payload['notice_type']
                         == "gcn.notices.icecube.lvk_nu_track_search"
                     ):
-                        if payload.get("pval_bayesian", 1) > 0.05:
+                        # 3 sigma
+                        if payload.get("pval_bayesian", 1) > 0.003:
```

Comment:
the exactly 0 is also there on purpose? Just checking.

Reply:
went from .05 to .003

Reply:
@Theodlz yes 3 sigma :)